stream-applications-build/pom.xml

```diff
@@ -49,6 +49,7 @@
 		<test-containers.version>1.16.3</test-containers.version>
 		<maven-flatten-plugin.version>1.2.5</maven-flatten-plugin.version>
 		<maven-surefire-plugin.version>2.22.2</maven-surefire-plugin.version>
+		<mock-server.version>5.13.2</mock-server.version>
```

Comment:
I am preparing to make all apps use same mock server version as there are several different versions being used currently.